packages/orbit-design-tokens/output/docs-tokens.json

```diff
@@ -11800,6 +11800,8 @@
   },
   "countryFlagShadow": {
     "type": "box-shadow",
+  "drawerOverlayBackground": {
```

Comment:
This is incorrect. Probably a rebase problem. Building the tokens again should fix it